app/assets/javascripts/discourse/components/highlight-search.js

```diff
@@ -1,12 +1,12 @@
 import Component from "@ember/component";
```

Comment:
Renaming `highlight-text` to `highlight-search` will break at least two themes in our repos. Is there a way you can add backwards compatibility with a deprecation?

Reply:
sure :+1:

Reply:
@eviltrout @vinothkannans Just a heads up, this broke a theme component for me today, previously `highlight-text` was in `discourse/lib/highlight-text`, now it is in `discourse/component/highlight-text`.

Reply:
@pmusaraj previously, `highlight-text` was in both `lib` and `component` folders. I added backward compatibility only for the component file since we're using it in ember templates. I will add backward compatibility for lib file too if it's going to break more than one theme/plugin.